src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -162,6 +164,11 @@ private function registerReceivers(ContainerBuilder $container)
     {
         $receiverMapping = array();
         foreach ($container->findTaggedServiceIds($this->receiverTag) as $id => $tags) {
+            $receiverClass = $container->findDefinition($id)->getClass();
+            if (!is_subclass_of($receiverClass, ReceiverInterface::class)) {
+                throw new RuntimeException(sprintf('Receiver class "%s" must implement interface "%s".', $receiverClass, ReceiverInterface::class));
```

Comment:
Can you add more context to the exception? I suggest:
```
Invalid receiver "{id}": class "{class}" must implement interface "{interface}"
```

Reply:
Better, thanks.